msg/de_DE.utf8/csql.msg

```diff
@@ -52,6 +52,7 @@ gültigen Optionen:\n\
       --no-trigger-action      deaktivieren Trigger Aktion\n\
       --delimiter=ARG          Trennzeichen zwischen Spalte (funktioniert nur mit -q)\n\
       --enclosure=ARG          Gehäuse für eine Ergebniszeichenfolge (funktioniert nur mit -q)\n\
+      --page-server            connect the page server available locally\n\
```

Comment:
Do not mention the option in csql help. The csql option --page-server option is for debugging only and should be hidden from users. Fix for all languages.